src/database/queries/reviews.js

```diff
@@ -14,19 +14,27 @@ exports.findAll = async (filter, attributes = {}) => {
 		return error
 	}
 }
-exports.countDistinct = async (filter, attributes = {}) => {
+
+exports.countDistinct = async (filter, attributes = ['resource_id']) => {
 	try {
-		const res = await reviews.count({
+		const res = await reviews.findAndCountAll({
 			where: filter,
-			distinct: true,
+			attributes: attributes, // Select the specified attributes, defaulting to 'resource_id'
+			distinct: true, // Ensure distinct results
```

Comment:
@adithyadinesh0412 this function can be generic like findAndCountAll then pass all the requirement as a params. So this we can use as generic

Reply:
changed the query to handle this @priyanka-TL